environments/production/terraform.yml

```diff
@@ -110,6 +110,17 @@ servers:
     group: "couchdb2_proxy"
     os: ubuntu_pro_bionic
 
+  - server_name: "couch11-production"
```

Comment:
Not a big deal, but curious why 11 and not 15 which would be the logical next number to pick.

Reply:
@dannyroberts we just wanted it to be in serial. Instead of 12,13,15,16... if we want to have 11,12,13...